crates/tendermint/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name        = "tendermint"
+description = """
+              Tendermint is a high-performance blockchain consensus engine that powers
+              Byzantine fault tolerant applications written in any programming language.
+              This crate provides types for representing information about Tendermint
+              blockchain networks, including chain IDs, block IDs, and block heights.
+              """
+version     = "0.0.0" # Also update `html_root_url` in lib.rs when bumping this
+authors     = ["Tony Arcieri <tony@iqlusion.io>"]
+license     = "Apache-2.0"
+homepage    = "https://www.tendermint.com/"
+repository  = "https://github.com/tendermint/kms/tree/master/crates/tendermint"
+readme      = "README.md"
+categories  = ["cryptography", "database"]
+keywords    = ["blockchain", "bft", "consensus", "cosmos", "tendermint"]
+
+[badges]
+circle-ci = { repository = "tendermint/kms" }
+
+[dependencies]
+chrono = { version = "0.4", features = ["serde"] }
+digest = "0.8"
+failure = "0.1"
+failure_derive = "0.1"
+ripemd160 = "0.8"
+serde = { version = "1", optional = true }
+serde_derive = { version =  "1", optional = true }
+subtle-encoding = "0.2"
+tai64 = { version = "1", optional = true, features = ["chrono"] }
+
+[features]
+default = ["serializers", "tai64"]
+serializers = ["serde", "serde_derive"]
```

Comment:
I was thinking of using JSON for the blob of network-related information we store on the YubiHSM2, so having `Serialize` and `Deserialize` implemented for all of these types is really handy...